components/pedigree/resources/src/main/resources/pedigree/view/studySelectionDialog.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * The UI Element for displaying prompts to the user in a movable/semi-transparent dialogue for study selection
+ *
+ * @class OkCancelDialogue
+ */
+define([], function(){
+  var StudySelectionDialog = Class.create( {
+
+      initialize: function(studies) {
+          var _this = this;
+
+          this.studies = studies;
+
+          var mainDiv = new Element('div', {'class' : "study-selection-dialog"});
+          var container = new Element('div', {'class' : 'xform'});
+
+          mainDiv.insert(container);
+
+          var submitButton = new Element('input', {type : 'button', value : "Submit", 'class' : 'button'});
```

Comment:
I would say the button should be named `Select` and not `Submit`.

Less important, but maybe `Cancel` can also be renamed to something more explicit like "cancel patient creation", to make it clear that cancel not just dismissed the studies dialog, but the entire patient creation procedure?